### ISSUE TITLE
Doesn't work in OBS for Linux

### ISSUE THREAD

**Issue by bmn001** [insin/obs-bounce#3]
In both v1.1 and 1.2 of the script, using OBS 27.0.1 on Ubuntu Linux, the bouncing image will start in the middle of the screen, head to the top-left, and then bounce repeatedly against the top left corner of the viewport.

**Comment by bmn001** [comment]
Some updates, in case anyone is still watching this script-

I was seeing this behavior with a 1280x1280 image.

When I resized it to 500x500, the bouncing effect works, but it will never hit the bottom of the screen.  It's as if there's an invisible barrier covering the bottom of the screen.

If I drag to shrink the image within OBS, the bouncing will work, but it then has the effect of raising that invisible barrier and the bouncing is limited to the top half of the screen.  If you drag to enlarge the image beyond its native dimensions, it will travel outside of the viewable area and bounce back in.

So basically, you can't really resize the image that you're animating.  The file has to contain the correct dimensions from the get-go.

Even then, there doesn't seem to be any randomness in the DVD bounce.  Once I had tweaked everything and got this "working", the image still bounces on the exact same parts of the frame, every single time.

**Comment by PlexSheep** [comment]
I have successfully gotten it to work on Debian Stable/12 in 2025. `OBS Studio - 29.0.2.1-1+b1 (linux)`. No problems here.
